openmaptiles/mbtile_tools.py

```diff
@@ -261,8 +262,8 @@ async def generate(self, tileset, reset, auto_minmax,
             f'Connecting to PostgreSQL at {pghost}:{pgport}, db={dbname}, user={user}...')
         try:
             async with asyncpg.create_pool(
-                database=dbname, host=pghost, port=pgport, user=user,
-                password=password, min_size=1, max_size=1,
+                    database=dbname, host=pghost, port=pgport, user=user,
+                    password=password, min_size=1, max_size=1,
```

Comment:
Is this additional indentation intentional?

Reply:
Yes, parameters to a function should be indented more than a block.  When i did a file reformat, these got changed.